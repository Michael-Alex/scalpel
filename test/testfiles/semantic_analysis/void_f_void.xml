<namespace id="n0">
 <simple_function name="f">
  <return_type>
   <fundamental_type type="void"/>
  </return_type>
 </simple_function>
</namespace>
