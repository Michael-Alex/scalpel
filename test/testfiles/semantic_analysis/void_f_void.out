{
  "id": "namespace-0",
  "simple functions":
  [
    {
      "id": "simple-function-0",
      "name": "f",
      "return type":
      {
        "fundamental type": "void"
      }
    }
  ]
}
